docs/source/configuration/overview.mdx

```diff
@@ -403,6 +403,21 @@ See [Tracing in the Apollo Router](./tracing/).
 Automatic Persisted Queries (APQ) enable GraphQL clients to send a server the _hash_ of their query string, _instead of_ the query string itself. This can significantly reduce network usage for very large query strings.
 The Apollo Router automatically supports APQ via its in-memory cache. See the [caching documentation](./caching) for related options.
 
+Subgraph requests support automatic persisted queries) by default. It can be deactivated with the `apq` option:
+
+```yaml title="router.yaml"
+supergraph:
+  apq:
+    subgraph:
+      # override for all subgraphs
+      all:
+        enabled: false
+      # override per subgraph
+      subgraphs:
+        products:
+          enabled: true
+```
+
 For more information on APQ, including client configuration, see [this article](/apollo-server/performance/apq/).
```

Comment:
I think this section of the documentation might need to be re-worked a bit: but as a question, how does the APQ optional #2386 behavior (the client-facing APQ, not the subgraph facing APQ) get disabled?

Reply:
I think it's weird to have `subgraph` configuration under a `supergraph` field. It's not really consistent with what we have currently. @BrynCooke any thoughts/suggestions ?

Reply:
@abernix dev was not merged yet in this PR, now the option for client facing APQ is there

Reply:
@bnjjj it feels like `apq` should be a top level option instead, with `supergraph` and `subgraph` sections

Reply:
Agree